vars/default_vars.yml

```diff
@@ -103,9 +103,12 @@ dhcpd_enabled: False
 named_install: True
 named_enabled: True
 block_DNS: False
+# Captive Portal highly experimental as of July 2018: https://github.com/iiab/iiab/pull/870
+py_captive_portal_install: True
+py_captive_portal_enabled: False
```

Comment:
Can we place these 3 lines into our [local_vars.yml](https://github.com/iiab/iiab/tree/master/vars) files as well?
```
# Captive Portal highly experimental as of July 2018: https://github.com/iiab/iiab/pull/870
py_captive_portal_install: True
py_captive_portal_enabled: False
```